.github/workflows/ci-ember-flight-icons.yml

```diff
@@ -30,18 +30,11 @@
 
 name: CI - Ember Flight Icons
 
-defaults:
-  run:
-    working-directory: ember-flight-icons
```

Comment:
Took another pass at this in https://github.com/hashicorp/flight/pull/153. I think removing this line might have caused problems in this PR as this is still required for the tests to work correctly

Reply:
ah gotcha - thank you for the follow-up!